File: FetchLottie.java
import java.net.URI;
import java.net.http.HttpClient;
import java.net.http.HttpRequest;
import java.net.http.HttpResponse;
import java.util.regex.Matcher;
import java.util.regex.Pattern;

public class FetchLottie {
    public static void main(String[] args) throws Exception {
        HttpClient client = HttpClient.newHttpClient();
        HttpRequest request = HttpRequest.newBuilder()
                .uri(URI.create("https://lottiefiles.com/free-animation/chatgpt-atlas-animation-mockup-JAD74oUkdb"))
                .header("User-Agent", "Mozilla/5.0 (Windows NT 10.0; Win64; x64)")
                .build();
        
        HttpResponse<String> response = client.send(request, HttpResponse.BodyHandlers.ofString());
        String body = response.body();
        
        java.nio.file.Files.writeString(java.nio.file.Paths.get("lottie.html"), body);
    }
}

File: fetch_lottie.js
const url = "https://lottiefiles.com/free-animation/chatgpt-atlas-animation-mockup-JAD74oUkdb";
fetch(url, {
    headers: {
        "User-Agent": "Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/91.0.4472.124 Safari/537.36",
        "Accept": "text/html"
    }
}).then(res => res.text()).then(html => {
    const lottieMatch = html.match(/https:\/\/[^\s"'<>]+\.lottie/);
    const jsonMatch = html.match(/https:\/\/[^\s"'<>]+\.json/);
    console.log("Lottie URL:", lottieMatch ? lottieMatch[0] : "Not found");
    console.log("JSON URL:", jsonMatch ? jsonMatch[0] : "Not found");
}).catch(err => console.error(err));
